### PR TITLE
732 -Question confirmation piping

### PR DESCRIPTION
**Pull request by tgandrews**
### What is the context of this PR?
Initial question confirmation piping. This is reliant on #53 

### How to review 
1. Ensure you can view a survey with piped data in the question confirmation
1. The correct answers are pickable (this is not true as it does not allow those on from the current page)

### Dependencies
- [x] - #53 - Need the method `getPreviousAnswersForPage`

### CODE REVIEW

**Comment by tgandrews** [comment]
[Piping into playback confirmation title](https://trello.com/c/NZHx1ULL/732-piping-into-playback-confirmation-title)

File: eq-author-api/repositories/QuestionConfirmationRepository.js [hunk 1]
```diff
@@ -45,12 +49,28 @@ module.exports = knex => {
       .restore(id)
       .then(head);
 
+  const getPipingAnswers = async id => {
+    const { pageId } = await findById(id);
+    return PreviousAnswerStrategy(knex).getPreviousAnswersForPage({
+      id: pageId,
+      includeSelf: true,
```
Comment (samiwel) [inline eq-author-api/repositories/QuestionConfirmationRepository.js]: Perhaps I misunderstood how this flag works, but when I tested e2e i wasn't able to select answers on the page that the confirmation page hangs off. I set something up like this (note I created Q2 after Q1):

- S1
  - Q2
    - Answer 2
  - Q1
    - Answer 1
    - Confirmation page

On the confirmation page, I was able to pipe answer 1 but not answer 2. Is that expected?

Comment (tgandrews) [reply eq-author-api/repositories/QuestionConfirmationRepository.js]: No thats not intentional. The include self should include it in the list of pages. I'll look into it.

Comment (tgandrews) [reply eq-author-api/repositories/QuestionConfirmationRepository.js]: Is your branch up to date this was resolved and I cannot recreate it locally. I also have a cypress test that covers it.

Comment (samiwel) [reply eq-author-api/repositories/QuestionConfirmationRepository.js]: Apologies, looks like my branch didn't have the latest changes. After a pull this worked as expected for me. 👍

**Pull request merged**